src/features/game/components/MaxItems.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext, useState } from "react";
+import ReCAPTCHA from "react-google-recaptcha";
+
+import close from "assets/icons/close.png";
+import token from "assets/icons/token.gif";
+
+import { Context } from "../GameProvider";
+import { useActor } from "@xstate/react";
+import { Button } from "components/ui/Button";
+import { CONFIG } from "lib/config";
+import { ITEM_DETAILS } from "../types/images";
+import { InventoryItemName } from "../types/game";
+
+export const MaxItems: React.FC = () => {
+  const { gameService } = useContext(Context);
+  const [gameState] = useActor(gameService);
+  const [showCaptcha, setShowCaptcha] = useState(false);
+  const maxedItem = gameState.context.maxedItem as InventoryItemName | "SFL";
+
+  const onCaptchaSolved = async (captcha: string | null) => {
+    await new Promise((res) => setTimeout(res, 1000));
+
+    gameService.send("SYNC", { captcha });
+  };
+
+  const makeTitle = () => {
+    const regex = new RegExp(/^[aeiou]/gi);
+    const startsWithVowel = regex.test(maxedItem);
+    const indefiniteArticle = startsWithVowel ? "an" : "a";
+    const item = maxedItem === "SFL" ? maxedItem : maxedItem.toLowerCase();
+
+    return `Have you become ${indefiniteArticle} ${item} horder?!`;
```

Comment:
Could we simplify the `indefiniteArticle` here and say "Are you hoarding wood?"

Reply:
This would mean we would need to have some pluralise logic that we would need to maintain. This was what we talked about this morning. 

I will change to `Are you an/a {resource} hoarder?`. We can't really escape the grammar check here.